mesos_master/tests/test_check.py

```diff
@@ -54,3 +38,59 @@ def test_check(check, instance, aggregator):
     aggregator.assert_metric('mesos.framework.total_tasks')
     aggregator.assert_metric('mesos.role.frameworks.count')
     aggregator.assert_metric('mesos.role.weight')
+
+
+def test_default_timeout(check, instance):
+    # test default timeout
+    check = check({}, instance)
+    check._get_master_roles = lambda v, x: json.loads(read_fixture('roles.json'))
+    check._get_master_stats = lambda v, x: json.loads(read_fixture('stats.json'))
+    check._get_master_state = lambda v, x: json.loads(read_fixture('state.json'))
+    check.check(instance)
```

Comment:
To avoid setup duplication, this can be factored into a separate function, example:

```python
def create_check(init_config, instance):
    check = MesosMaster(common.CHECK_NAME, init_config, [instance])
    check._get_master_roles = lambda v, x: json.loads(read_fixture('roles.json'))
    check._get_master_stats = lambda v, x: json.loads(read_fixture('stats.json'))
    check._get_master_state = lambda v, x: json.loads(read_fixture('state.json'))
    return check
```